project-two/projects.html

```diff
@@ -0,0 +1,152 @@
+</head>
+<body>
+  <header>
+
+    <a class="header-name-title-container" href="index.html">
+      <div class="header-name-container">
+        <span class="header-name-square"></span>
+        <h2 class="header-name">Maya Nelson</h2>
+      </div>
+      <span class="header-name-slash">/</span>
+      <h3 class="header-title">project manager</h3>
+    </a>
+    
+    <nav class="nav-mobile">
+      <a class="nav-mobile-link" href="">
+        <span class="nav-mobile-burger-top-layer"></span>
+        <span class="nav-mobile-burger-middle-layer"></span>
+        <span class="nav-mobile-burger-bottom-layer"></span>
+      </a>
+    </nav>
+
+    <nav class="nav-desktop">
+      <ul class="nav-desktop-list">
+        <li class="nav-desktop-list-item">
+          <a class="nav-desktop-list-item-link nav-desktop-list-item-about-me" href="index.html">about me</a>
+        </li>
+        <li class="nav-desktop-list-item">
+          <a class="nav-desktop-list-item-link nav-desktop-list-item-resume" href="resume.html">resume</a>
+        </li>
+        <li class="nav-desktop-list-item">
+          <a class="nav-desktop-list-item-link nav-desktop-list-item-projects" href="projects.html">projects</a>
+        </li>
+        <li class="nav-desktop-list-item">
+          <a class="nav-desktop-list-item-link nav-desktop-list-item-contact" href="contact.html">contact</a>
+        </li>
+      </ul>
+    </nav>
+
+  </header>
+
+  <main>
+
+    <div class="main-container">
+    
+      <div class="projects-heading-container">
+        <span class="projects-heading-square"></span>
+        <h1 class="projects-heading">Projects</h1>
+      </div>
+      <p class="projects-heading-paragraph">I'm a paragraph. Click here to add your own text and edit me. It's easy. Just click “Edit Text” or double click me to add your own content and make changes to the font. I'm a great place for you to tell a story and let your users know a little more about you.</p>
+
+      <div class="projects-container">
+        
+        <section class="project-container">
+          <div class="project-text-container">
+            <div class="project-name-role-container">
+              <h2 class="project-name">Project name 01</h2>
+              <span class="project-role">Role Title</span>
+            </div>
+            <p class="project-description">I'm a paragraph. Click here to add your own text and edit me. It's easy. Just click “Edit Text” or double click me to add your own content and make changes to the font. I'm a great place for you to tell a story and let your users know a little more about you.</p>
+          </div>
+            <picture class="project-image-container">
+              <source srcset="images/leaflet-mobile.webp" media="(max-width: 750px)">
+              <source srcset="images/leaflet-desktop.webp" media="">
+              <img class="project-image" src="images/leaflet-mobile.webp" alt="A technology conference leaflet">
+            </picture>
```

Comment:
This is really good use of the picture element for change of art direction here instead of an img element with object-fit cover